Dockerfile

```diff
@@ -10,6 +10,7 @@ WORKDIR /app
 COPY --from=build-image ./app/dist ./dist
 COPY package* ./
 COPY migrations ./migrations
-RUN npm ci --production
+COPY migrateUpWithWrapper.mjs ./migrateUpWithWrapper.mjs
```

Comment:
`migrateUpWithWrapper.mjs` is a script that is needed to perform the db migration **BEFORE** running the app (`users-api`).
It is not a Typescript program and is not part of the service codebase so to speak. That is why it lives outside of the `dist` folder.